src/main/java/frc/robot/commands/TestAll.java

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.constants.IntakeConstants;
+import frc.robot.subsystems.AimingSubsystem;
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.LauncherSubsystem;
+import frc.robot.subsystems.SwerveSubsystem;
+
+public class TestAll extends Command {
+  IntakeSubsystem mIntake;
+  AimingSubsystem mAiming;
+  LauncherSubsystem mLauncher;
+  SwerveSubsystem mSwerve;
+  boolean mTestIntakeSeperately;
+  boolean mTestAimingSeperately;
+  boolean mTestLauncherSeperately;
+  boolean mTestSwerveSeperately;
+
+  public TestAll(IntakeSubsystem intake, AimingSubsystem aiming, LauncherSubsystem launcher, SwerveSubsystem swerve,
+  boolean testIntakeSeperately, boolean testAimingSeperately, boolean testLauncherSeperately, boolean testSwerveSeperately) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    mIntake = intake;
+    mAiming = aiming;
+    mLauncher = launcher;
+    mSwerve = swerve;
+
+    mTestIntakeSeperately = testIntakeSeperately;
+    mTestAimingSeperately = testAimingSeperately;
+    mTestLauncherSeperately = testLauncherSeperately;
+    mTestSwerveSeperately = testSwerveSeperately;
+  }
+  
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    if (mTestIntakeSeperately) {
+      testIntake();
+    }
+    if (mTestAimingSeperately) {
+      testAim();
+    }
+    if (mTestLauncherSeperately) {
+      testLauncher();
+    }
+    if (mTestSwerveSeperately) {
+      testSwerve();
+    }
+    testCycle();
+  }
+
+  public void testCycle() {
+    // Intake Portion
+    mSwerve.setChassisSpeed(1, 0, 0, false,false); 
```

Comment:
Where is all of this run? There aren't any commands, just a function that sets stuff. This will run in one cycle and finish